### PR TITLE
[github] Add labeler for pull-request

### PR DESCRIPTION
**Pull request by lvyanquan**
Add an action to auto label pr with connector type based on the files it changed for user searching.
For example:

截屏2023-10-19 上午11 55 48

### CODE REVIEW

**Comment by FlechazoW** [comment]
I think it would be better to use lowercase.

**Comment by lvyanquan** [comment]
address it.

**Review by FlechazoW — APPROVED** [review]
Nice job!

**Review by gtk96 — APPROVED** [review]
+1

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (leonardBang) [inline .github/labeler.yml:2]: Could you add copyright info when you add header?  and you can also add header for other `yml` files

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+test:
+  - flink-cdc-e2e-tests/**/*
+  - flink-connector-test-util/**/*
+db2:
+  - flink-connector-db2-cdc/**/*
+  - flink-sql-connector-db2-cdc/**/*
+mongodb:
+  - flink-connector-mongodb-cdc/**/*
+  - flink-sql-connector-mongodb-cdc/**/*
+mysql:
```
Comment (leonardBang) [inline .github/labeler.yml:43]: Could you refer the latest module structure? I think we should have `mysql-cdc-connector` and `mysql-pipeline-connector`

**Review by leonardBang — COMMENTED** [review]
Thanks @lvyanquan for the contribution, sorry for the late review. I left some comments.

File: .github/labeler.yml (hunk 2 above)
Comment (lvyanquan) [reply .github/labeler.yml]: mysql-pipeline-connector is not added yet, do we need to add label for this and other connectors in advance?

Comment (leonardBang) [reply .github/labeler.yml]: Good catch, we can add them later

**Review by leonardBang — APPROVED** [review]
Thanks @lvyanquan for the update, LGTM

**Pull request merged**